File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/PlusApiResultListAchievementVO.kt
package com.sdkwork.app

data class PlusApiResultListAchievementVO(
    val data: List<AchievementVO>? = null,
    val code: String? = null,
    val msg: String? = null,
    val requestId: String? = null,
    val ip: String? = null,
    val hostname: String? = null,
    val errorName: String? = null
)

File: sdkwork-app-sdk-typescript/src/types/plus-api-result-list-achievement-vo.ts
import type { AchievementVO } from './achievement-vo';

/** API调用结果 */
export interface PlusApiResultListAchievementVO {
  /** Response data */
  data: AchievementVO[];
  /** Response code: 2000=success, 4xxx=business failure, 5xxx=server error */
  code: string;
  /** Business message */
  msg: string;
  /** Request identifier */
  requestId: string;
  /** Client IP address */
  ip?: string;
  /** Server hostname */
  hostname?: string;
  /** Business error name */
  errorName: string;
}
